RR.App.Tests/Controllers/HRIS/EmployeeBankingControllerIntegrationTests.cs

```diff
@@ -170,7 +170,7 @@ public async Task Get_ReturnsOkResult()
     [Fact]
     public async Task CreateReadUpdateDeleteEmployeeBanking_ReturnsOkResult()
     {
-        var employeeDto = EmployeeTestData.EmployeeDtoNew;
+        var employeeDto = EmployeeTestData.EmployeeDto6;
```

Comment:
why the 6??

Reply:
Its the same test data it used beforeI just renamed the testdata because I also added other ones to that file. But I can always just change it back.

Reply:
Changed it back. Tests still passed.